docs/user-guide/kubectl-overview.md

```diff
@@ -33,160 +33,271 @@ Documentation for other releases can be found at
+##### Example
 
-Please use `kubectl help [command]` for detailed information about a command.
+To print a list of pods sorted by name, you run:
 
-Please refer to [kubectl](kubectl/kubectl.md) for a complete list of available commands and flags.
+`$ kubectl get pods --sort-by=.metadata.name`
 
-## Common Operations
+## Examples: Common operations
 
-For explanation, here I gave some mostly often used `kubectl` command examples. Please replace sample names with actual values if you would like to try these commands.
+Use the following set of examples to help you familiarize yourself with running the commonly used `kubectl` operations:
 
-1. `kubectl create` - Create a resource by filename or stdin
+ * `kubectl create` - Create a resource using a file or stdin.
 
-		// Create a service using the data in example-service.yaml.
+		// Create a service using the definition in example-service.yaml.
 		$ kubectl create -f example-service.yaml
 
-		// Create a replication controller using the data in example-controller.yaml.
+		// Create a replication controller using the definition in example-controller.yaml.
 		$ kubectl create -f example-controller.yaml
 
-		// Create objects whose definitions are in a directory. This looks for config objects in all .yaml, .yml, and .json files in <directory> and passes them to create.
+		// Create the objects that are defined in any .yaml, .yml, or .json file within the <directory> directory.
 		$ kubectl create -f <directory>
 
-2. `kubectl get` - Display one or many resources
+ * `kubectl get` - List one or more resources.
 
-		// List all pods in ps output format.
+		// List all pods in plain-text output format.
 		$ kubectl get pods
 
-		// List all pods in ps output format with more information (such as node name).
+		// List all pods in plain-text output format and includes additional information (such as node name).
 		$ kubectl get pods -o wide
 
-		// List a single replication controller with specified name in ps output format. You can use the alias 'rc' instead of 'replicationcontroller'.
+		// List the replication controller with the specified name in plain-text output format. Tip: You can shorten and replace the 'replicationcontroller' resource type with the alias 'rc'.
 		$ kubectl get replicationcontroller <rc-name>
 
-		// List all replication controllers and services together in ps output format.
+		// List all replication controllers and services together in plain-text output format.
 		$ kubectl get rc,services
 
-3. `kubectl describe` - Show details of a specific resource or group of resources
+ * `kubectl describe` - Display all the details for one or more resources.
```

Comment:
I would change "all the details" to "detailed state". To actually get *all* details, you  have to get the raw object (`kubectl get <type/name> -o yaml|json`)